src/index.ts

```
@@ -185,29 +183,37 @@ export class Engine {
                     binaryPath = resolve(__dirname, '../../../node_modules', this.binary);
                 }
 
-                const childOptions = {
-                    'env': Object.assign({'ENGINE_CONFIG': JSON.stringify(childConfig)}, process.env),
-                    'maxBuffer': 5 * 1024 * 1024
```

Hmm, was adding this along with switching to spawn just a mistake?

---

Yep; it was added it back when this code called `exec()` - it was silently ignored since we switched to `spawn()`.